apps/yapms/src/lib/components/modals/ModalBase.svelte

```diff
@@ -27,7 +29,7 @@
 </script>
 
 <dialog class="modal modal-bottom lg:modal-middle" bind:this={dialog} on:close={close}>
-	<div class="modal-box flex flex-col w-full">
+	<div class="modal-box flex flex-col w-full" class:!max-w-full={fullWidth}>
```

Comment:
Here I am allowing the modal to grow to the full size of the screen if fullWidth is set.